erizoAPI/WebRtcConnection.cc

```diff
@@ -595,8 +595,11 @@ NAUV_WORK_CB(WebRtcConnection::promiseResolver) {
       ELOG_WARN("%s, message: Resolving promise with no valid value, using empty string", obj->toLog());
       resolver->Resolve(Nan::GetCurrentContext(), Nan::New("").ToLocalChecked()).IsNothing();
     }
+    persistent->Reset();
+    delete persistent;
     obj->futures.pop();
     obj->Unref();
+    v8::Isolate::GetCurrent()->RunMicrotasks();
```

Comment:
Nice! Is this safe to use in all the places we resolve promises? I'm thinking about `MediaStream` particularly.

Reply:
you're right, I'm looking for all places where we use v8::Promise